schemes/dark/color-background.json

```diff
@@ -72,6 +72,9 @@
             },
             "stronger": {
               "value": "{global.color.primary.01.value}"
+            },
```

Comment:
Since `genesys-tokens@6.0.0` (already in use in every brand) adding or removing tokens in brands also requires a change in https://github.com/DevoInc/genesys-tokens/tree/master/tokens/src/tokens/brand-defaults, that should be ideally done in advance. The tokens inside `brand-default` are a set of default values that concrete brands override partially or totally. 

Having a default brand inside tokens helps us with type declarations and ensure consistency among

Reply:
To Jose's question, right now there is no test, besides the fact that the type declarations will fail when trying to access the new tokens. I'll look into it and see if there is any chance of having a test.

Reply:
Pull Request created in genesys-tokens with the corresponding changes --> https://github.com/DevoInc/genesys-tokens/pull/42